modules/changing-the-log-level.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * cli/debugging-issues.adoc
+:_content-type: PROCEDURE
+[id="changing-the-log-level_{context}"]
+= Changing the log level
+
+.Prerequisites
+
+* You have configured the `ROX_ENDPOINT` environment variable.
+** Set the `ROX_ENDPOINT` environment variable:
++
+[source,terminal,subs="+quotes"]
+----
+$ export ROX_ENDPOINT=_<host:port>_ <1>
+----
++
+<1> The host and port information that you want to store in the  `ROX_ENDPOINT` environment variable.
```

Comment:
Since we have this now duplicated multiple times, does it make sense to make this a module and just import it wherever we need? In case of changes we have a single place to change then instead of multiple places.

Reply:
I have created a new module for setting environment variables